docs(i18n): contributing guidelines in polish (#37350)
freeCodeCamp/freeCodeCamp: docs/i18n-languages/polish/CONTRIBUTING.md

# Wytyczne dotyczące twojego wkładu w FreeCodeCamp

Cześć. 👋

> funkcjonowanie freeCodeCamp.org jest możliwe dzięki tysiącom życzliwych wolontariuszy takich jak Ty. Jesteśmy wdzięczni za Twój wkład i cieszymy się, że możemy Cię powitać na pokładzie.

Reviewer: ```suggestion
Funkcjonowanie freeCodeCamp.org jest możliwe dzięki tysiącom życzliwych wolontariuszy takich jak Ty. Jesteśmy wdzięczni za Twój wkład i cieszymy się, że możemy powitać Cię na pokładzie.
```

Ściśle egzekwujemy nasz [„Kodeks postępowania”](https://www.freecodecamp.org/code-of-conduct). Poświęć chwilę, aby go przeczytać. Ma tylko 196 słów.

> Szczęsliwego rozwijania kodu open source!

Reviewer: "Szczęśliwego"

> ## Oto kilka zabawnych sposobów, w które możesz pomóc

Reviewer: ```suggestion
## Oto kilka fajnych sposobów, w które możesz pomóc
```

> Możesz przyczynić się do jednego z tych:

Reviewer: "[...] do jednego z tych rzeczy:"

> 1. [Przyczyniaj się do tej otwartej źródłowej bazy danych](#https://github.com/freeCodeCamp/freeCodeCamp/). Pomóż nam edytować wyzwania związane z kodowaniem lub naprawiaj błędy na platformie edukacyjnej.

Reviewer: ```suggestion
1. [Rozwijaj tę otwartoźródłowej bazy danych](#https://github.com/freeCodeCamp/freeCodeCamp/). Pomóż nam edytować wyzwania związane z kodowaniem lub naprawiaj błędy na platformie edukacyjnej.
```
This is the proper translation of "open-source" https://sjp.pwn.pl/poradnia/haslo/otwartozrodlowy;10114.html

> 1. [Przyczyniaj się do tej otwartej źródłowej bazy danych](#https://github.com/freeCodeCamp/freeCodeCamp/). Pomóż nam edytować wyzwania związane z kodowaniem lub naprawiaj błędy na platformie edukacyjnej.

Author: wymagana poprawka do tej poprawki- masz tu literówkę

> 1. [Przyczyniaj się do tej otwartej źródłowej bazy danych](#https://github.com/freeCodeCamp/freeCodeCamp/). Pomóż nam edytować wyzwania związane z kodowaniem lub naprawiaj błędy na platformie edukacyjnej.

Reviewer: Corrected too.

> 1. [Przyczyniaj się do tej otwartej źródłowej bazy danych](#przyczyniaj się do tej otwartej źródłowej bazy danych). Pomóż nam edytować wyzwania związane z kodowaniem lub naprawiaj błędy na platformie edukacyjnej.. Pomóż nam edytować wyzwania związane z kodowaniem lub naprawiać błędy na platformie edukacyjnej.

Reviewer: ```suggestion
1. [Przyczyniaj się do tej otwartej źródłowej bazy danych](#https://github.com/freeCodeCamp/freeCodeCamp/). Pomóż nam edytować wyzwania związane z kodowaniem lub naprawiaj błędy na platformie edukacyjnej.
```

> 2. Pomoc dla kamperów na naszym [forum publicznym](https://www.freecodecamp.org/forum/). [Odpowiedz na pytania dotyczące kodowania](https://www.freecodecamp.org/forum/?max_posts=1) lub [podaj im informacje zwrotne na temat ich projektów kodowania](https://www.freecodecamp.org/forum/c/project-feedback?max_posts=1).

Reviewer: ```suggestion
2. Pomoc dla kamperów na naszym [publicznym forum](https://www.freecodecamp.org/forum/). [Odpowiedz na pytania dotyczące kodowania](https://www.freecodecamp.org/forum/?max_posts=1) lub [podaj im informacje zwrotne na temat ich projektów kodowania](https://www.freecodecamp.org/forum/c/project-feedback?max_posts=1).
```

> 3. Pomóż nam dodawać napisy (podpisy zamknięte) do naszych [Filmów na kanale YouTube](https://www.youtube.com/channel/UC8butISFwT-Wl7EV0hUK0BQ/videos).

Reviewer: ```suggestion
3. Pomóż nam dodawać napisy do naszych [filmów na kanale YouTube](https://www.youtube.com/channel/UC8butISFwT-Wl7EV0hUK0BQ/videos).
```

> ## Przyczyniaj się do tej otwartej źródłowej bazy danych.

Reviewer: ```suggestion
## Przyczyniaj się do tej otwartoźródłowej bazy danych.
```

Nasza baza danych zawiera tysiące [wyzwań związanych z kodowaniem](https://learn.freecodecamp.org) oraz inne kody źródłowe, które zasilają naszą platformę edukacyjną.

##### Twórz, aktualizuj i naprawiaj błędy w naszych wyzwaniach związanych z kodowaniem.

> Wszystkie nasze wyzwania związane z kodowaniem są obsługiwane przez społeczność, dostarczającą specjalistyczną wiedzę od wolontariuszy takich jak Ty.

Reviewer: ```suggestion
Wszystkie nasze wyzwania związane z kodowaniem są obsługiwane przez społeczność dostarczającą specjalistyczną wiedzę od wolontariuszy takich jak Ty.
```

Możesz pomóc je rozszerzyć i poprawić ich brzmienie. Możesz również aktualizować historie użytkowników, aby lepiej wyjaśnić koncepcję lub usunąć zbędne i poprawić testy wyzwań, aby lepiej testowały kod ludzi.

**Jeśli jesteś zainteresowany poprawą tych wyzwań związanych z kodowaniem, oto [jak pracować nad wyzwaniami związanymi z kodowaniem](/docs/how-to-work-on-coding-challenges.md).**

##### Pomóż nam naprawić błędy w platformie edukacyjnej freeCodeCamp.org

Nasza platforma edukacyjna działa na nowoczesnym stosie JavaScript. Posiada różne komponenty, narzędzia i biblioteki, w tym między innymi Node.js, MongoDB, LoopBack, OAuth 2.0, React, Gatsby, Webpack i wiele innych.

Ogólnie rzecz biorąc,

- Mamy serwer API oparty na Node.js.
- Zestaw aplikacji klienckich opartych na React.
- Skrypt, którego używamy do oceny naszych projektów front-end.

> rozwijanie naszego stacku wymaga trochę zrozumienia API, ES6 Syntax, i chęci nauki.

Reviewer: ```suggestion

Rozwijanie naszego stacku wymaga trochę zrozumienia API, ES6 Syntax, i chęci nauki.
```

Zasadniczo oczekujemy podstawowej znajomości niektórych z wyżej wymienionych technologii, narzędzi i bibliotek. Biorąc to pod uwagę, nie jesteście zobowiązani do bycia ekspertem w tym zakresie, aby móc wnieść swój wkład.

> **Jeśli chcesz pomóc nam udoskonalić naszą bazę danych, możesz albo [skonfigurować freeCodeCamp lokalnie](/docs/how-to-setup-freecamp-locally.md) albo użyć Gitpod, darmowego środowiska online dev.**

Reviewer: ```suggestion
**Jeśli chcesz pomóc nam udoskonalić naszą bazę danych, możesz albo [skonfigurować freeCodeCamp lokalnie](/docs/how-to-setup-freecamp-locally.md) albo użyć Gitpod, darmowego internetowego środowiska deweloperskiego.**
```

[![Otwórz w Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/freeCodeCamp/freeCodeCamp)

> (Uruchamia gotowe do kodowania środowisko dev dla freecodecamp w przeglądarce.)

Reviewer: ```suggestion
(Uruchamia gotowe do kodowania środowisko deweloperskie dla freecodecamp w przeglądarce.)
```

## Najczęściej zadawane pytania

### Jak mogę zgłosić błąd, którego nie ma na pokładzie?

> Jeśli uważasz, że znalazłeś błąd, przeczytaj najpierw artykuł ["Help I've Found a Bug"](https://forum.freecodecamp.org/t/how-to-report-a-bug/19543) i postępuj zgodnie z jego instrukcjami.

Reviewer: ```suggestion
Jeśli uważasz, że znalazłeś błąd, przeczytaj najpierw artykuł ["Pomocy znalazłem błąd"](https://forum.freecodecamp.org/t/how-to-report-a-bug/19543) i postępuj zgodnie z jego instrukcjami.
```

Other links have polish names too, so I suppose this one should have as well?

Jeśli jesteś pewien, że jest to nowy błąd, stwórz nowy problem z GitHub. Upewnij się, że dołączysz jak najwięcej informacji, abyśmy mogli odtworzyć błąd. Mamy predefiniowany szablon problemu, który pomoże Ci w jego rozwiązaniu.

> Proszę zwrócić uwagę, że wszelkie problemy, które szukają pomocy przy kodowaniu w przypadku wyzwania, zostaną zamknięte. Śledzenie spraw jest ściśle związane z zagadnieniami związanymi z bazą danych i dyskusjami. W razie wątpliwości, przed sporządzeniem raportu należy [szukać pomocy na forum](https://www.freecodecamp.org/forum).

Reviewer: ```suggestion
Zwróć uwagę, że wszelkie problemy, które szukają pomocy przy kodowaniu w przypadku wyzwania, zostaną zamknięte. Śledzenie spraw jest ściśle związane z zagadnieniami związanymi z bazą danych i dyskusjami. W razie wątpliwości, przed sporządzeniem raportu należy [szukać pomocy na forum](https://www.freecodecamp.org/forum).
```

##### Jak mogę zgłosić problem bezpieczeństwa?

> Proszę nie tworzyć problemów z GitHub w sprawach bezpieczeństwa. Zamiast tego, wyślij e-mail na adres `security@freecodecamp.org` i natychmiast się temu przyjrzymy.

Reviewer: ```suggestion
Proszę nie tworzyć problemów z GitHub w sprawach bezpieczeństwa. Zamiast tego wyślij e-mail na adres `security@freecodecamp.org` i natychmiast się temu przyjrzymy.
```

> Proszę nie tworzyć problemów z GitHub w sprawach bezpieczeństwa. Zamiast tego, wyślij e-mail na adres `security@freecodecamp.org` i natychmiast się temu przyjrzymy.

Reviewer: ```suggestion
Proszę nie tworzyć problemów z GitHub w sprawach bezpieczeństwa. Zamiast tego wyślij e-mail na adres `security@freecodecamp.org` a natychmiast się temu przyjrzymy.
```

##### Co oznaczają te różne etykiety, które są oznaczone na tematach?

Nasi moderatorzy społeczności [triage](https://en.wikipedia.org/wiki/Software_bug#Bug_management) zgłaszają problemy i wnioski w oparciu o ich priorytet, dotkliwość i inne czynniki. Możesz [znaleźć kompletny słowniczek ich znaczeń tutaj](https://github.com/freecodecamp/freecodecamp/labels).

> Powinieneś przejść przez [**`help wanted`**](https://github.com/freeCodeCamp/freeCodeCamp/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) lub [**`first timers welcome`**](https://github.com/freeCodeCamp/freeCodeCamp/issues?q=is%3Aopen+is%3Aissue+label%3A%22first+timers+welcome%22) w celu szybkiego przeglądu tego, nad czym możesz pracować. 

Reviewer: ```suggestion
Powinieneś przejść przez [**`potrzebna pomoc`**](https://github.com/freeCodeCamp/freeCodeCamp/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) lub [**`nowicjusze mile widziani`**](https://github.com/freeCodeCamp/freeCodeCamp/issues?q=is%3Aopen+is%3Aissue+label%3A%22first+timers+welcome%22) w celu szybkiego przeglądu tego, nad czym możesz pracować. 
```

> Powinieneś przejść przez [**`help wanted`**](https://github.com/freeCodeCamp/freeCodeCamp/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) lub [**`first timers welcome`**](https://github.com/freeCodeCamp/freeCodeCamp/issues?q=is%3Aopen+is%3Aissue+label%3A%22first+timers+welcome%22) w celu szybkiego przeglądu tego, nad czym możesz pracować. 

Reviewer: ```suggestion
Powinieneś przejść przez [**`potrzebna pomoc`**](https://github.com/freeCodeCamp/freeCodeCamp/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) lub [**`nowicjusze mile widziani`**](https://github.com/freeCodeCamp/freeCodeCamp/issues?q=is%3Aopen+is%3Aissue+label%3A%22first+timers+welcome%22) w celu szybkiego przeglądu tego, nad czym możesz pracować. 
```

> Są one do pobrania i nie musisz ubiegać się o pozwolenie przed rozpoczęciem pracy nad nimi. Jeśli te kwestie nie są jasne, co należy zrobić, nie krępuj się zadawać pytań w komentarzach.

Reviewer: ```suggestion
Są one do pobrania i nie musisz ubiegać się o pozwolenie przed rozpoczęciem nad nimi pracy. Jeśli w kwestiach tych nie jest jasne, co należy zrobić, nie krępuj się zadawać pytań w komentarzach.
```

> Są one do pobrania i nie musisz ubiegać się o pozwolenie przed rozpoczęciem pracy nad nimi. Jeśli te kwestie nie są jasne, co należy zrobić, nie krępuj się zadawać pytań w komentarzach.

Reviewer: ```suggestion
Są one do pobrania i nie musisz ubiegać się o pozwolenie przed rozpoczęciem nad nimi pracy. Jeśli w kwestiach tych nie jest jasne, co należy zrobić, nie krępuj się zadawać pytań w komentarzach.
```

#### Znalazłem literówkę, czy powinienem zgłosić problem, zanim złożę prośbę o ciągnięcie?

> W przypadku literówek i innych zmian w słownictwie, możesz bezpośrednio otwierać żądania ciągnięcia bez wcześniejszego tworzenia problemu. Kwestie są bardziej przydatne do omówienia większych problemów związanych z kodem lub aspektami strukturalnymi programu nauczania.

Reviewer: ```suggestion
W przypadku literówek i innych zmian w słownictwie, możesz bezpośrednio otwierać żądania ciągnięcia bez wcześniejszego tworzenia problemu. Problemy są bardziej przydatne do omówienia większych problemów związanych z kodem lub aspektami strukturalnymi programu nauczania.
```

> W przypadku literówek i innych zmian w słownictwie, możesz bezpośrednio otwierać żądania ciągnięcia bez wcześniejszego tworzenia problemu. Kwestie są bardziej przydatne do omówienia większych problemów związanych z kodem lub aspektami strukturalnymi programu nauczania.

Reviewer: ```suggestion
W przypadku literówek i innych zmian w słownictwie, możesz bezpośrednio otwierać żądania ciągnięcia bez wcześniejszego tworzenia problemu. Problemy są bardziej przydatne do omówienia większych problemów związanych z kodem lub aspektami strukturalnymi programu nauczania.
```

##### Jak mogę dostać problem przypisany do mnie?

> Zazwyczaj nie przydziela się problemów nikomu innemu niż długoletni współpracownicy, aby uniknąć ambitnego nie pojawienia się na zajęciach. Zamiast tego postępujemy zgodnie z poniższą polityką, aby być sprawiedliwym dla wszystkich:

Reviewer: ```suggestion
Zazwyczaj nie przydziela się problemów nikomu innemu niż długoletnim współpracownikom, aby uniknąć ambitnego nie dostarczenia wyników. Zamiast tego postępujemy zgodnie z poniższą polityką, aby być sprawiedliwym dla wszystkich:
```

1. Preferowane jest połączenie pierwszego wniosku o wyciągnięcie ręki w każdej sprawie.
2. W przypadku wielokrotnych próśb o ten sam problem, dajemy pierwszeństwo jakości kodu w próśb o wyciągnięcie. 
   - Czy dołączyłeś testy? 
   - Czy złapałeś wszystkie przypadki?
   - Czy upewniłeś się, że wszystkie testy przeszły pomyślnie i potwierdziłeś, że wszystko działa lokalnie?
3. W końcu popieramy prośby o pociągnięcie, które są zgodne z naszymi zaleceniami.
 - Podążał pan za listą kontrolną? 
 - Czy podałeś nazwę swojej listy kontrolnej w sposób znaczący?

> Nie potrzebujesz żadnego zezwolenia dla spraw, które są oznaczone jako "pomoc poszukiwana" lub "pierwsze powitanie czasowe", jak wyjaśniono wcześniej. Postępuj zgodnie z wytycznymi i otwórz prośbę o pomoc.

Author: literówka "potrzeban pomoc"

> Nie potrzebujesz żadnego zezwolenia dla spraw, które są oznaczone jako "pomoc poszukiwana" lub "pierwsze powitanie czasowe", jak wyjaśniono wcześniej. Postępuj zgodnie z wytycznymi i otwórz prośbę o pomoc.

Reviewer: Great catch, thanks.

##### Jestem nowy w GitHubie i Open Source, od czego powinienem zacząć?

> Przeczytaj nasz [How to Contribute to Open Source Guide](https://github.com/freeCodeCamp/how-to-contribute-to-open-source).

Reviewer: ```suggestion
Przeczytaj nasz [Poradnik jak wnieść wkład do Open Source](https://github.com/freeCodeCamp/how-to-contribute-to-open-source).
```

> ##### Utknęłam na czymś, co nie jest zawarte w tej dokumentacji. Jak mogę uzyskać pomoc?

Reviewer: ```suggestion
##### Utknąłem na czymś, co nie jest zawarte w tej dokumentacji. Jak mogę uzyskać pomoc?
```

Możesz poprosić o pomoc:

- [Kategoria "Współtwórcy" na naszym forum publicznym](https://www.freecodecamp.org/forum/c/contributors)

> - [Nasz publiczny współpracownik - chat room na Gitterze](https://gitter.im/FreeCodeCamp/Contributors)

Reviewer: ```suggestion
- [Nasz publiczny chat room dla współpracowników na Gitterze](https://gitter.im/FreeCodeCamp/Contributors)
```

> Jesteśmy podekscytowani, że pomożemy Ci wziąć udział we wszystkich tematach, nad którymi chciałbyś pracować. Zachęcamy do zadawania nam pytań na tematy pokrewne, a my chętnie udzielimy wyjaśnień. Upewnij się, że szukasz swojego zapytania przed wysłaniem nowego. Bądź uprzejmy i cierpliwy. Nasza społeczność wolontariuszy i moderatorów jest zawsze w pobliżu, aby poprowadzić Cię przez Twoje pytania.

Reviewer: "Jesteśmy podekscytowani, by pomóc Ci ze wzięciu udziału"

> Jeśli masz wątpliwości, możesz skontaktować się z naszym zespołem programistów platformy, aby Ci w tym pomóc:

Author: done
